### DISCUSSION TITLE
YAML for model responce

### DISCUSSION THREAD

**Discussion by KSemenenko** [microsoft/semantic-kernel#3236] [Ideas]
I have a theory than YAML will take less tokens than JSON,
If so, is it possible that using this format in requests will make them cheaper?
And according to my survey, json sometimes turns out to be invalid.

**Comment by KSemenenko** [comment]
and we can use https://github.com/hadashiA/VYaml to make it faster

**Comment by madsbolaris** [comment]
From my own experience, the models have an _much_ easier time outputting YAML because it doesn't need to worry about closing tags and escaping characters like quotes. I also agree that YAML would use fewer tokens.
